ci/steps/build.sh

```diff
@@ -10,52 +10,38 @@ cd "$(git rev-parse --show-toplevel)/ci/steps"
 
```

Comment:
Would be good if we got rid of this script entirely in favour of it being inlined in the makefile